internal/splunk/common.go

```diff
@@ -63,8 +81,11 @@ func (e *Event) GetMetricValues() map[string]any {
 
 	values := map[string]any{}
 	for k, v := range e.Fields {
-		if strings.HasPrefix(k, "metric_name:") {
-			values[k[12:]] = v
+		// only consider metric name if it fits regex criteria.
+		// use matches[1] since first element contains entire string.
+		// first subgroup will be the actual metric name.
```

Comment:
should this comment be in the `getMetricNameFromField` function?

Reply:
will move

Reply:
moved